docs/monitoring/index.md

```diff
@@ -1,65 +1,72 @@
 
-In parallel, an MLRun job runs, reading the parquet files and performing drift analysis. The drift analysis data is stored so 
-that the user can retrieve it in the Iguazio UI or in a Grafana dashboard.
+Read also how {ref}`model monitoring supports the gen AI model server<genai-mmonitor>`.
 
-When you enable model monitoring, you effectively deploy three components:
-- application controller function: handles the monitoring processing and the triggers the apps that trigger the writer. The controller is a scheduled batch job whose frequency is determined by `base_period`. 
-- stream function: monitors the log of the data stream. It is triggered when a new log entry is detected. The monitored data is used to create real-time dashboards, detect drift, and analyze performance.
-- writer function: writes to the database and outputs alerts.
+## Streaming platforms and credentials
+
+Model monitoring supports open-source streaming platforms such as Kafka, TDEngine, MySQL (8.0 and higher), in addition to integration with the Iguazio AI platform V3IO data layer. 
+Before you deploy the model monitoring or serving function, you need to {py:meth}`set the credentials <mlrun.projects.MlrunProject.set_model_monitoring_credentials>`. 
+
+## Model monitoring applications
+
+When you call `enable_model_monitoring` on a project, by default MLRun deploys te onitoring app, `HistogramDataDriftApplication`, which is 
+tailored for classical ML models (not LLMs, gen AI, deep-learning models, etc.). It includes:
+* Total Variation Distance (TVD) &mdash; The statistical difference between the actual predictions and the model's trained predictions.
+* Hellinger Distance &mdash; A type of f-divergence that quantifies the similarity between the actual predictions, and the model's trained predictions.
+* Kullback–Leibler Divergence (KLD) &mdash; The measure of how the probability distribution of actual predictions is different from the second model's trained reference probability distribution.
+
+You can create your own model monitoring applications, for LLMs, gen AI, deep-learning models, etc., based on the class {py:meth}`mlrun.model_monitoring.applications.ModelMonitoringApplicationBaseV2`. 
+
+Projects are used to group functions that use the same model monitoring application. You first need to create a project for aspecific application. 
+Then you disable the default app, enable your customer app, and create and run the functions. 
+
+See the:
+- User flow example **in ??????**.
+- Custom apps in **?????**.
+
+The basic flow for classic ML and other models is the same, but the apps and the infer requests are different.
+
+
+
+
+## Model and model monitoring endpoints 
+
+Each unique combination of runtime function (a deployed Nuclio function) and model has an endpoint and a corresponding {py:meth}`model endpoint <mlrun.model_monitoring.api.get_or_create_model_endpoint>`. 
```

Comment:
Suggsest not to use the informal "you". 
Is there a section about the UI?

Reply:
@assaf758 the entire doc addresses the reader in the first person "you".
UI is described peripherally in monitoring/index.html#model-and-model-monitoring-endpoints. Were you thinking of a section devoted to the UI?

Reply:
While model monitoring is configured in the SDK, it largely "consumed" from the UI.. So yes, I think a UI section is needed - for model-monitoring at least describing the general structure: selecting model-endpoint, then selecting metrics/results to show..